http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -323,9 +350,13 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
 
 }
 
-final case class AKSInterpreterConfig(terraAppSetupChartConfig: TerraAppSetupChartConfig,
-                                      coaAppConfig: CoaAppConfig,
-                                      aadPodIdentityConfig: AadPodIdentityConfig,
-                                      appRegistrationConfig: AzureAppRegistrationConfig,
-                                      samConfig: SamConfig
+final case class AKSInterpreterConfig(
+  terraAppSetupChartConfig: TerraAppSetupChartConfig,
+  coaAppConfig: CoaAppConfig,
+  aadPodIdentityConfig: AadPodIdentityConfig,
+  appRegistrationConfig: AzureAppRegistrationConfig,
+  samConfig: SamConfig,
+  pollingConfig: PollingConfig = PollingConfig(maxAttempts = 120, delay = 10 seconds, interruptAfter = 20 minutes)
```

Comment:
Does this seem alright or should I have these be environment variables/.conf values?